config/toml.go

```diff
@@ -21,6 +21,8 @@ const DefaultConfigTemplate = `# This is a TOML config file.
 
 log-level = "{{ .BaseConfig.LogLevel }}"
 oracle-mnemonic = "{{ .BaseConfig.OracleMnemonic }}"
+acc-num = "{{ .BaseConfig.AccNum }}"
+acc-index = "{{ .BaseConfig.AccIndex }}"
```

Comment:
For unity, how about prefixing oracle like this:
```suggestion
oracle-acc-num = "{{ .BaseConfig.AccNum }}"
oracle-acc-index = "{{ .BaseConfig.AccIndex }}"
```

Reply:
Good. agree with you. I'll change it